projects/vdk-plugins/vdk-impala/src/vdk/plugin/impala/templates/load/fact/insert-snapshot/README.md

```diff
@@ -0,0 +1,51 @@
+### Purpose:
+
+This template can be used to load raw data from Data Lake to target 'Snapshot Periodic Fact Table' in Data Warehouse.
```

Comment:
I think this is misleading a bit. The template doesn't happen snapshot of records. It simply appends records. 

I would also rename it to "insert" or "append"